common/src/test/java/com/revenuecat/purchases/common/EntitlementInfoTest.kt

```diff
@@ -18,6 +18,10 @@ import kotlin.time.Duration.Companion.days
 @Config(manifest = Config.NONE)
 class EntitlementInfoTest {
 
+    private val oneDayAgo = 1.days.ago()
+    private val twoDaysAgo = 2.days.ago()
+    private val oneDayFromNow = 1.days.fromNow()
```

Comment:
Dates are only created one now to prevent _slightly_ off timestamps causing equal tests to fail